db/schema.rb

```diff
@@ -371,10 +371,10 @@
   create_table "whitehall_migrations", force: :cascade do |t|
     t.text "organisation_content_id"
     t.text "document_type"
-    t.text "document_subtypes", array: true
     t.datetime "finished_at"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.text "document_subtypes", array: true
```

Comment:
Looks like this has slipped in on an unrelated commit?

Reply:
Honestly have no idea how this got in, I've not had to look at this :/ at any rate I've removed it.